docs/examples/3rdparty_analysis_workflow.sh

```diff
@@ -89,6 +89,12 @@ datalad update alice --merge
 datalad install result.txt
```

Comment:
should there be some commit message, so it looks like a lab book, e.g.

datalad install --msg "Opened the black hole, TODO: find the close toggle" result.txt